content/docs/authorization/Migration guide from v1 to v2/_index.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Authorization - v2 Migration guide
+linktitle: Migration guide from v1 to v2
+weight: 1
+description: >
+  CSM for Authorization v1 to v2 Migration guide
+---
+
+CSM for Authorization v2 has major architectural changes due to which user will not be able to upgrade CSM for Authorization v1 to CSM for Authorization v2. This page provide a referent guide to user **How to migrate** from version 1 to version 2 release.
+
+**Before migration please note following points**
+  - CSM for Authorization v2 calculate the actual usage of capacity provision by syncing with array.
+  - Volumes belong to tenant are identified using **Volume Prefix** configured in tenant CR.
+  - Volumes without **Volume Prefix** will not be accounted in usage capacity calculation as ownership of volume is unknown with the volume prefix.
+  - User should rename all volumes that are needed to be accounted with **Volume Prefix** before migration to v2.
+
+## Storage Systems
+
+In CSM for Authorization v1 setup, list the storage to get all the storages configured in the environment.
+Example: :
+
+```
+karavictl storage list --admin-token admintoken.yaml --addr csm-authorization.host.com
+
+{
+  "storage": {
+    "powerflex": {
+      "3000000000011111": {
+        "Endpoint": "https://1.1.1.1",
+        "Insecure": true,
+        "Password": "(omitted)",
+        "User": "admin"
+      }
+    }
+  }
+}
+```
+In CSM for Authorization v2 storage are created using CRDs, each storage that is created in v1 environment should also be created at v2 environment, here is an example above storage:
```

Comment:
Can you update this to "For each Storage in a v1 environment, create Storage using the Storage CRD. Example: "